axiom/nr_distributed_trace.c

```diff
@@ -1164,6 +1165,8 @@ char* nr_distributed_trace_create_w3c_tracestate_header(
   const char* app_id;
   char* trace_context_header = NULL;
   char* sampled = "0";
+  double priority;
```

Comment:
The return value of `nr_distributed_trace_get_priority` is `nr_sampling_priority_t`:
```suggestion
  nr_sampling_priority_t priority;
```